pallets/deitos/src/types.rs

```diff
@@ -53,17 +55,24 @@ pub enum PaymentPlanPeriods {
 #[derive(Clone, Encode, Decode, Eq, PartialEq, Debug, MaxEncodedLen, TypeInfo)]
 #[scale_info(skip_type_params(T))]
 #[codec(mel_bound(T: pallet::Config))]
-pub struct InfraProviderDetails<T: pallet::Config> {
-    // Price per block
-    pub price_per_block: BalanceOf<T>,
+pub struct IPDetails<T: pallet::Config> {
+    /// Total IP Storage
+    pub total_storage: Storage,
```

Comment:
type name `Storage` look a bit confusing to me:
1. it can be misinterpreted as a runtime storage
2. I don't understand the units (Kb, Mb, Gb, etc.)